bin/mc.cpp

```diff
@@ -83,7 +83,7 @@ MiniMC::Host::ExitCodes mc_main (MiniMC::Model::Controller& controller, const Mi
   messager.message("Finished Reachability");
   
   
-  if (verdict == MiniMC::Algorithms::Reachability::Verdict::Found) {
+  if (verdict.getVerdict() == MiniMC::Algorithms::Reachability::Verdict::Verdict::Found) {
     MiniMC::Support::getMessager ().message (MiniMC::Support::Localiser ("Found Violation").format ());
     std::cerr << reach.foundState () << std::endl;
```

Comment:
Since the `Verdict`-object contains the found state, then the output should use instead of the "cached" state of the reachability algorithm .
Since results are now kept in  a result-object then the cached state in the reachability algorithm should be removed completely (I think)